backend/routes/user_routes.py

```diff
@@ -0,0 +1,51 @@
+from os import environ
+from fastapi import APIRouter, HTTPException, Depends
+from sqlmodel import Session, select
+from uuid import uuid4
+from datetime import timedelta
+
+from models import User
+from core.database import get_session
+from utils.jwt_handler import create_jwt_token
+
+router = APIRouter()
+webdomain = environ.get("WEBDOMAIN")
+
+
+@router.post("/login")
+async def send_login_link(
+    phone_number: int, session: Session = Depends(get_session)
+):
```

Comment:
Before using the user input, validate it to ensure it is a clean, globalised E.164 number. Regex: `r'^\+[1-9]\d{1,14}$'`.

Reply:
https://github.com/akvo/rag-doll/pull/12/commits/41b5b0c8e43281fbcce4f64961b575b9549cb9f4